lib/net/networking_engine.js

```diff
@@ -838,7 +838,7 @@ shaka.net.NetworkingEngine.NumBytesRemainingClass = class {
  * download, and allows the request to be aborted if the network is slow.
  *
  * @implements {shaka.extern.IAbortableOperation.<shaka.extern.Response>}
- * @extends {shaka.util.AbortableOperation}
+ * @extends {shaka.util.AbortableOperation.<shaka.extern.Response>}
```

Comment:
do we even need this `@extends`? Class extends it directly anyway, `@implements` should give all info about types we need.

Reply:
btw, we shouldn't have `.<>` here, but I guess ESLint plugin handle this only for native types...